app/src/main/java/me/moallemi/multinavhost/MainActivity.kt

```diff
@@ -15,12 +15,11 @@ class MainActivity : AppCompatActivity(), BottomNavigationView.OnNavigationItemS
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-
+        if (savedInstanceState == null) tabManager.currentController = tabManager.navHomeController
```

Comment:
please do not use single line `if` 

```suggestion
        if (savedInstanceState == null) {
            tabManager.currentController = tabManager.navHomeController
        }
```